client/src/elm/Translate.elm

```diff
@@ -23496,13 +23496,13 @@ translationSet trans =
                 SymptomBabyTiresQuicklyWhenFeeding ->
                     { english = "Baby tires quickly when feeding"
                     , kinyarwanda = Just "Umwana ahita ananirwa iyo atangiye kurya"
-                    , kirundi = Nothing
+                    , kirundi = Just "Uruyoya ruruha vuba igihe rwonka"
                     }
 
                 SymptomCoughingOrTearingWhileFeeding ->
                     { english = "Coughing/tearing while feeding (<6 months)"
                     , kinyarwanda = Just "Gukorora/Kwiriza iyo atangiye kurya (munsi y'amezi 6)"
-                    , kirundi = Nothing
+                    , kirundi = Just ""
```

Comment:
Why this change?

Reply:
I might have missed it I will revert that

Reply:
Reverted myself.